sdkv2.0.0/sdkv2.0.0/MojioSDK/MojioClient.swift

```diff
@@ -210,6 +210,30 @@ class MojioClient: NSObject {
         return self
     }
     
+    func query (top : String?, skip : String?, filter : String?, select : String?, orderby : String?) -> Self {
+        
+        self.REQUEST_URL = self.REQUEST_URL + "?"
+        
+        if top != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$top=" + top!
+        }
+        if skip != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$skip=" + skip!
+        }
+        if filter != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$filter=" + filter!
+        }
+        if select != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$select=" + select!
+        }
+        if orderby != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$orderby=" + orderby!
+        }
+
```

Comment:
Another issue is that we are not encoding the URL parameters which is generally good practice. We should be using parameters (dictionary) when calling Alamofire.request instead of building out our own query strings.